tests/test_consent/api/test_views.py

```diff
@@ -127,7 +110,6 @@ def _assert_expectations(self, response, expected_body, expected_status):
             },
             400
         ),
-        # No consent in an enterprise course enrollment nor in an audit.
```

Comment:
The issue for me with the comments is they are hard to maintain. In the end the test cases are just covering all possible permutations of a certain set of values related to DSC and DSC's enterprise customer.